docs/_docs/quick-start/cpp.adoc

```diff
@@ -51,30 +44,62 @@ For information about the {cpp} thin client, see link:thin-clients/cpp-thin-clie
 On unix systems, you can use the command line to build and run the examples included in the Ignite distribution.
 
 === Prerequisites
-The following packages should be installed:
-- C++ compiler
-- libssl
-- autotools
-- automake
-- libtool
+include::includes/cpp-linux-build-prerequisites.adoc[]
 
 === Building C++ Ignite
 
-. Download and unzip the Ignite binary release into a directory. We'll refer to this as `{IGNITE_HOME}`.
-. `cd {IGNITE_HOME}/platforms/cpp`
-. `libtoolize && aclocal && autoheader && automake --add-missing && autoreconf`
-. `./configure`
-. `make`
-
-=== Running the Thick Client Example
-
-. `cd {IGNITE_HOME}/platforms/cpp/examples`
-. `libtoolize && aclocal && autoheader && automake --add-missing && autoreconf`
-. `./configure`
-. `cd put-get-example`
-. `make`
-. `./ignite-put-get-example`
-
+- Download and unzip the Ignite binary release into a directory. We'll refer to this as `{IGNITE_HOME}`.
+- Create build directory for cmake. We'll refer to this as `{CPP_BUILD_DIR}`
+- Do the folowing:
```

Comment:
"Use the commands below to build and install Ignite C++" - or you can use a shorter version. The point is that I would say that the reader needs to "use the commands" to do X.